step-capstone/src/components/Utilities/SuggestionPopup.js

```diff
@@ -0,0 +1,191 @@
+import React from "react";
+import {
+    Slide,
+    Paper,
+    Tabs,
+    Tab,
+    Grid,
+    Box,
+    Button,
+    CircularProgress,
+    IconButton
+} from "@material-ui/core"
+import CloseIcon from '@material-ui/icons/Close';
+import "../../styles/SuggestionPopup.css"
+import PreferenceForm from "./PreferenceForm"
+import { handleSuggestions } from "../../scripts/Suggestions"
+
+export default class SuggestionPopup extends React.Component {
+    constructor(props) {
+        super(props);
+        this.state = {
+            userPref: this.props.userPref,
+            coordinates: this.props.coordinates,
+            timeRange: this.props.timeRange,
+            items: this.props.items,
+            tabPos: 0,
+            foodSuggestions: [],
+            activitySuggestions: [],
+            suggestionsLoaded: false,
+            radius: this.props.radius
+        }
+
+        this.handleUserPrefChange = this.handleUserPrefChange.bind(this);
+        this.handleToggleTab = this.handleToggleTab.bind(this);
+        this.renderSuggestions = this.renderSuggestions.bind(this);
+        this.getSuggestions = this.getSuggestions.bind(this);
+    }
+
+    componentDidMount() {
+        // this.getSuggestions();
```

Comment:
Nit: Please remove or replace with a TODO. If this function should do nothing, there should be some sort of pass you can use.